src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  if (days <= 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return (40 * days);
+  }
 
+  if (((days > 3) || (days === 3)) && (days < 7)) {
+    return (40 * days - 20);
+  }
+
+  if ((days > 7) || (days === 7)) {
+    return (40 * days - 50);
+  }
```

Comment:
Good job!
But lets make some variables for values, because for now we don't understand what it is - 40, 50, 20 ...